source/documentation/values/calculations.html.md.erb

```diff
@@ -258,3 +303,81 @@ combined at build time, so `min(12px % 10, 10%)` will throw an error.
     padding-right: 20px;
   }
 <% end %>
+
+### [`round()`]
+<% impl_status dart: "1.64.0", libsass: false, ruby: false, feature: "Special function syntax" do %>
+  LibSass, Ruby Sass, and versions of Dart Sass prior to 1.64.0 *always* parse
+  `round()` a as Sass function. To create a plain CSS calculation function
+  call for those implementations, you can write something like
+  `round(#{$number})` instead.
+
+  [special functions]: ../syntax/special-functions
+<% end %>
+
+The `round(<strategy>, value, step)` function accepts an optionl rounding strategy, 
+a value to be rounded and a rounding interval `step`. 
+`strategy` should be `nearest`, `up`, `down`, or `to-zero`.
+`round(value)` is also supported for backwards compatibility.
+
+<% example do %>
+  $number: 12.5px;
+  $step: 15px;
+
+  .post {
+    // Since these round() calls are valid calculation expressions, they're
+    // parsed as calculations.
+    padding-left: round(nearest, $number, $step);
+    padding-right: round($number + 10px);
+    padding-bottom: round($number + 10px, $step + 10%);
+  }
+
+  ===
+  $number: 12.5px;
+  $step: 15px;
+
+  .post
+    // Since these round() calls are valid calculation expressions, they're
+    // parsed as calculations.
+    padding-left: round(nearest, $number, $step)
+    padding-right: round($number + 10px)
+    padding-bottom: round($number + 10px, $step + 10%)
+
+  ===
+  .post {
+    padding-left: 15px;
+    padding-right: 23px;
+    padding-bottom: round(22.5px, 15px + 10%);
+  }
+<% end %>
+
+### [`abs()`]
+<% impl_status dart: "1.64.0", libsass: false, ruby: false, feature: "Special function syntax" do %>
+  LibSass, Ruby Sass, and versions of Dart Sass prior to 1.64.0 *always* parse
+  `abs()` a as Sass function. To create a plain CSS calculation function
+  call for those implementations, you can write something like
+  `abs(#{$number})` instead.
+
+  [special functions]: ../syntax/special-functions
+<% end %>
+
+Given that CSS resolves percentages before calculations, 
+the global `abs(value)` function compatibiliy with % unit parameters is deprecated.
+In the future, this will emit a CSS abs() function to be resolved by the browser.
```

Comment:
This is definitely worth calling out, although it might be better in a `heads_up` block because it's a warning about a potential trap users could fall into. I find this phrasing pretty confusing, though: it's not immediately clear what "CSS resolves percentages before calculations" means in practice to stylesheet authors, and "compatibility with % unit parameters" is pretty abstract. Consider instead starting from a concrete example, explaining how it works in CSS, then explaining how it works in Sass and why it has to be different.